tasks/install_and_configure_texlive.yml

```diff
@@ -0,0 +1,17 @@
+- name: Install TeX Live
+  ansible.builtin.package:
+    name:
+      - texlive
+      - texlive-fonts-extra
+      - texlive-latex-extra
+      - texlive-science
+      - texlive-xetex
+
+- name: Create config file to increase size of texmf buffer
+  ansible.builtin.template:
+    dest: /etc/texmf/texmf.d/99buffer_size.cnf
+    group: root
+    mode: 0644
+    owner: root
+    src: buffer_size.cnf.j2
+  notify: Ensure that any texmf config changes take effect
```

Comment:
Should we be using the `role_name : handler_name` syntax mentioned in the [Ansible documentation](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_handlers.html#handlers-in-roles)?

Reply:
I can't for the life of me get this syntax to work.  I haven't been able to determine when this feature was added, but I suspect it is only available in newer versions of Ansible than we are currently using.  (We have Ansible pinned to `<6` in `requirements-test.txt`.)

Reply:
_So I dug into this_. The reason it doesn't work is because we have to include the role in the molecule configuration as `ansible-role-cyhy-reports` and this means that `ansible-role-cyhy-reports` is the role name propagated. You can see this [here](https://github.com/cisagov/ansible-role-cyhy-reports/actions/runs/4471739244/jobs/7857008560#step:7:346) with

```console
TASK [ansible-role-cyhy-reports : Create config file to increase size of texmf buffer] ***
```

and the handler similarly impacted [here](https://github.com/cisagov/ansible-role-cyhy-reports/actions/runs/4471739244/jobs/7857008560#step:7:400) with

```console
RUNNING HANDLER [ansible-role-cyhy-reports : Ensure that any texmf config changes take effect] ***
```

, but then role dependencies use the `role_name` of the installed role [here](https://github.com/cisagov/ansible-role-cyhy-reports/actions/runs/4471739244/jobs/7857008560#step:7:333) with

```console
TASK [ncats_webd : Install the ncats-webd package] *****************************
```

So the `notify` would have to be `cyhy_reports : Ensure that any texmf config changes take effect` to work correctly in usage but this would not work for our molecule testing configuration. In this (vaguely cursed) [branch](https://github.com/cisagov/ansible-role-cyhy-reports/tree/testing/adjust_role_name_for_notify) you can see what I'm talking about [here](https://github.com/cisagov/ansible-role-cyhy-reports/actions/runs/4691092555/jobs/8315084847#step:7:352) with

```console
TASK [cyhy_reports : Create config file to increase size of texmf buffer] ******
```

and the handler running as expected [here](https://github.com/cisagov/ansible-role-cyhy-reports/actions/runs/4691092555/jobs/8315084847#step:7:406) with

```console
RUNNING HANDLER [cyhy_reports : Ensure that any texmf config changes take effect] ***
```

Reply:
Great job running that to ground!